ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -208,21 +208,34 @@ export const useNewMhrRegistration = () => {
     MhRegistrationSummaryIF[] {
     const sortedDraftFilings = orderBy(mhrDrafts, ['createDateTime'], ['desc'])
 
-    const registerNewUnit = mhrHistory.filter(registrations =>
```

Comment:
Considering this composable is used in a few places, is it safe to update the logic here? Won't it break anything pre-existing?

Reply:
Glad you made me check. We're good. This is the only place that MhrHistory is being used at present.  I checked the api call and although I added the optional parameter, it is only being used here as well.